### PR TITLE
Add an example of the new Presto task

### PR DESCRIPTION
**Pull request by lu4nm3**

### CODE REVIEW

File: flytetester/app/workflows/presto_workflow.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+schema = Types.Schema([("a", Types.String), ("b", Types.Integer)])
+
+presto_task = SdkPrestoTask(
+    query="SELECT * FROM hive.city.fact_airport_sessions WHERE ds = '{{ds}}' LIMIT 10",
+    output_schema=schema.schema_type,
+    routing_group="{{ routing_group }}",
+    catalog="hive",
+    schema="city",
+    task_inputs=inputs(ds=Types.String, routing_group=Types.String),
```
Comment (EngHabu) [inline flytetester/app/workflows/presto_workflow.py]: Just a nit, do you mind moving this to be the first param of the task...

File: flytetester/app/workflows/presto_workflow.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+@workflow_class()
+class PrestoWorkflow(object):
+    ds = Input(Types.String, required=True, help="Test string with no default")
+    routing_group = Input(Types.String, required=True, help="Test string with no default")
+
+    p_task = presto_task(ds=ds, routing_group=routing_group)
+
+    output_a = Output(p_task.outputs.results, sdk_type=Types.Schema())
```
Comment (EngHabu) [inline flytetester/app/workflows/presto_workflow.py]: shouldn't this be:
```suggestion
    output_a = Output(p_task.outputs.results, sdk_type=schema)
```

File: flytetester/app/workflows/presto_workflow.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+from flytekit.sdk.tasks import inputs
+from flytekit.sdk.types import Types
+from flytekit.sdk.workflow import workflow_class, Input, Output
+from flytekit.common.tasks.presto_task import SdkPrestoTask
+
+schema = Types.Schema([("a", Types.String), ("b", Types.Integer)])
+
+presto_task = SdkPrestoTask(
+    query="SELECT * FROM hive.city.fact_airport_sessions WHERE ds = '{{ds}}' LIMIT 10",
+    output_schema=schema.schema_type,
```
Comment (wild-endeavor) [inline flytetester/app/workflows/presto_workflow.py]: Matt says: change this to `Types.Schema()`

Comment (wild-endeavor) [reply flytetester/app/workflows/presto_workflow.py]: but can you verify afterwards, that the interface that the output writer sees still has all the right columns?

File: flytetester/app/workflows/presto_workflow.py [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+from flytekit.sdk.types import Types
+from flytekit.sdk.workflow import workflow_class, Input, Output
+from flytekit.common.tasks.presto_task import SdkPrestoTask
+
+schema = Types.Schema([("a", Types.String), ("b", Types.Integer)])
+
+presto_task = SdkPrestoTask(
+    task_inputs=inputs(ds=Types.String, routing_group=Types.String),
+    query="SELECT * FROM hive.city.fact_airport_sessions WHERE ds = '{{ds}}' LIMIT 10",
+    output_schema=schema.schema_type,
```
Comment (wild-endeavor) [inline flytetester/app/workflows/presto_workflow.py]:
```suggestion
    output_schema=schema,
```

**Comment by lu4nm3** [comment]
@wild-endeavor Thanks! I just pushed the last of the changes to make it match flytekit

**Pull request merged**